tsconfig.json

```diff
@@ -1,16 +1,14 @@
 {
   "compilerOptions": {
-    "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
-    "module": "esnext" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
-    "allowJs": true /* Allow javascript files to be compiled. */,
-    "declaration": false,
-    "strict": true /* Enable all strict type-checking options. */,
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
-    "skipLibCheck": true /* Skip type checking of declaration files. */,
-    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
-    "moduleResolution": "node",
-    "lib": ["dom", "es2017"],
-    "removeComments": false
-
-  }
-}
\ No newline at end of file
+    "target": "es5",
+    "module": "commonjs",
```

Comment:
Why do you prefer commojs to esmodules?

Reply:
I don't have a preference, that was the value from the config I copied. I changed it to `esnext`.